surface/src/3rdparty/opennurbs/opennurbs_archive.cpp

```diff
@@ -4194,7 +4194,14 @@ CUserDataHeaderInfo::CUserDataHeaderInfo()
 
 void CUserDataHeaderInfo::Initialize()
 {
-  memset(this,0,sizeof(*this));
+  m_classid = {};
+  m_itemid = {};
+  m_appid = {};
+  m_3dm_version = 0;
+  m_3dm_opennurbs_version = 0;
+  m_copycount = 0;
+  m_bLastSavedAsGoo = false;
+  m_xform.Zero();
 }
```

Comment:
I've initialized all member variables of `CUserDataHeaderInfo` to 0. And `m_classid, m_itemid, m_appid` are all `class ON_UUID` without customized constructors, so I set them using zero initialization.

https://github.com/PointCloudLibrary/pcl/blob/6555d8a6f10f0d2fa6f60e99e10360141ad57a89/surface/src/3rdparty/opennurbs/opennurbs_archive.cpp#L4173-L4188